src/components/courseDetail/SealModal.module.css

```
@@ -0,0 +1,124 @@
+  justify-content: center;
+  align-items: center;
+  z-index: 1000;
+}
+
+.modal {
+  position: relative;
+  width: 262px;
+  height: 311px;
+  background: white;
+  border-radius: 20px;
+  background: transparent;
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

background is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)